components/head-static.html

```diff
@@ -35,17 +35,20 @@
 <script src="js/webcomponents-bundle.js"></script>
 
 <!-- myuw-web-components -->
-<script type="module" src="https://unpkg.com/@myuw-web-components/myuw-app-bar@1.5.4/dist/myuw-app-bar.min.mjs"></script>
+<script type="module"
+  src="https://unpkg.com/@myuw-web-components/myuw-app-bar@1.5.4/dist/myuw-app-bar.min.mjs"></script>
 <script nomodule src="https://unpkg.com/@myuw-web-components/myuw-app-bar@1.5.4/dist/myuw-app-bar.min.js"></script>
 
 <script type="module" src="https://unpkg.com/@myuw-web-components/myuw-banner@1.1.1/dist/myuw-banner.min.mjs"></script>
 <script nomodule scr="https://unpkg.com/@myuw-web-components/myuw-banner@1.1.1/dist/myuw-banner.min.js"></script>
 
+<script type="module" src="http://cdn.my.wisc.edu/@myuw-web-components/myuw-help@1.5.1/myuw-help.min.mjs"></script>
```

Comment:
It turns out this needs to be via `https:`` rather than `http:`; fixed in #942 .